src/ComponentFolders/CareersPage/CareersPageMain.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import ContentA from './ContentA';
+import ContentB from './ContentB';
+import ContentC from './ContentC';
+
+function CareersPageMain() {
+  return (
+    <div className='h-screen w-screen'>
+      <div className='flex flex-col font-poppins w-screen '>
+        <div >
+             <ContentA />
+            
+        </div>
+        <div className='bg-yellow-300 w-full'>
```

Comment:
Great job, can you just changes the background color to "bg-amber-200" to suit the rest of the site 💯